setup.py

```diff
@@ -39,6 +39,7 @@ def grep(attrname):
 
 install_requires = [
     'Flask>=0.7',
+    'werkzeug<1',
```

Comment:
can I suggest to pin to `werkzeug>=0.15.0,<1` to leave less room for interpretation by pips greedy dependency resolving @NDevox 
```suggestion
    'werkzeug~=0.15',
```

Reply:
scratch that just saw your comment

Reply:
😄 it was a good suggestion though.